app/konnect/analytics/generate-reports.md

```diff
@@ -32,34 +32,36 @@ To set up a new report, open {% konnect_icon analytics %} [Analytics](https://cl
 
 1. Choose a time frame to display.
 
-    You can choose a preset time frame ranging from 1 day to 1 year, or
-    select **Custom** to set any date range.
+    Select between **Relative** or **Custom** to set any date range.
 
-    All time frames are static. The report will capture a snapshot of data
-    in whichever time frame you choose, but the range always stops at the time
-    that the report was created. You can see a preview of the exact range below
+    **Relative** time frames are dynamic and the report will capture a snapshot of data
+    relatively from the time a user views the report.
+
+    **Custom** time frames are static and the report will capture a snapshot of data
+    in whichever time frame you choose. You can see a preview of the exact range below
     the time frame selector. For example:
 
     ```
     Sunday, Jun 12, 9:57:45 AM (CEST) to Monday, Jun 13, 9:57:45 AM (CEST)
     ```
 
 
-1. Choose a report type.
+1. Choose an entity type.
```

Comment:
Is this being renamed in the UI (because the field is currently "Report type")?

Reply:
Yes.